### ISSUE TITLE
Coordinate jump on scrolling

### ISSUE THREAD

**Issue by NeilFraser** [google/blockly#420]
* Open Firefox (Chrome works fine)
* Load tests/generators/index.html on develop branch.
* Click 'Load' to import a bunch of blocks for logic tests.
* Scroll to the bottom of the page using the scrollbar.
* Use the mouse to click on the workspace and drag up or down.

I observe the workspace jumping, and the vertical scrollbar displacing below the bottom of the screen.

**Comment by picklesrus** [comment]
I think I figured out what's going on except from the part where it works in chrome... I'm going to guess you had two different window sizes that happened to make one of the resizes not necessary.  I can repro something similar with copy/paste in chrome.

What I think is happening is that domToWorkspace calls domToBlock which triggers a contents resize, but then it calls moveBy and then no resize contents gets triggered.  The cheap thing to do is to call resize at the end of domToWorkspace.  But thinking a tiny bit harder, I found copy/paste is also a problem for a similar reason so I'm going to send you a PR with a change to moveBy
